storyblok/storyblok-design-system
# feat(work-120): add new icon for workflow stage

Added a new icon for workflow stage

## Pull request type

Jira Link: [WORK-120](https://storyblok.atlassian.net/browse/WORK-120)




## How to test this PR

1. open https://storyblok-design-system-git-feat-work-120-6743fc-storyblok-com.vercel.app/?path=/story/design-system-components-sbicon--all-icons
2. make sure that there are a new icon called `workflow-status`

## What is the new behavior?



- Must have a new icon listed

![Screen Shot 2022-10-21 at 06 23 00](https://user-images.githubusercontent.com/7618411/197161954-2f9ef3f4-d6e1-4f6c-9ecc-b3b5ea7d6a2e.png)


## Other information
